os/binfmt/libelf/libelf_cache.c

```diff
@@ -447,23 +447,22 @@ int elf_cache_init(int filfd, uint16_t offset, off_t filelen, uint8_t compressio
 	binfo("filfd: %d offset: %d filelen: %d compression_type: %d\n", filfd, offset, filelen, compression_type);
 
 	/* Initialize the ELF params */
-	number_blocks_caching = CONFIG_ELF_CACHE_BLOCKS_COUNT;
 	cache_blocks_size = CONFIG_ELF_CACHE_BLOCK_SIZE;
 	file_len = filelen;
 	number_of_blocks = file_len / cache_blocks_size;
 	elf_compress_type = compression_type;
 
+	/* Extra unaligned data apart from blocksize */
+	if (file_len % cache_blocks_size)
```

Comment:
Needs brackets

Reply:
> Is this missing part of [2b7c427](https://github.com/Samsung/TizenRT/commit/2b7c427cc42bc63113caf3d8e25b0dfc4d73fce5)?

Yes, it was a mistake from engineer end, round off 10% was not taken care.